pkg/controller/directvolumemigration/migrate.go

```diff
@@ -38,9 +37,8 @@ func (r *ReconcileDirectVolumeMigration) migrate(ctx context.Context, direct *mi
 		Phase:            direct.Status.Phase,
 		PhaseDescription: direct.Status.PhaseDescription,
 		PlanResources:    planResources,
-		MigrationUID:     string(migration.UID),
-
-		Tracer: r.tracer,
+		MigrationUID:     migrationUID,
```

Comment:
If the MigrationUID is only used in one place, it feels like maybe we should just use `GetMigrationForDVM()` to grab it the Migration later on instead of putting the UID in this struct. No strong feelings either way though.

Reply:
@djwhatle Fixed